src/commands/database.rs

```diff
@@ -12,6 +12,10 @@ use crate::question;
 pub async fn create(cli: &mut Connection, options: &CreateDatabase, _: &Options)
     -> Result<(), anyhow::Error>
 {
+    if cli.get_version().await?.specific().major >= 5 {
+        anyhow::bail!("'edgedb database create' is deprecated in EdgeDB 5+. Please use 'edgedb branch create'");
```

Comment:
Oh, tests are failing because this is a hard error here.

You probably meant `eprintln!()`

Reply:
yeah I was gonna change lol